client/src/components/ActivityList/ActivityList.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap');
+
+.ActivityList {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 0px 40px;
+}
+
+.activitiesContainer {
```

Comment:
We want to have consistent text formatting for our CSS classes. Could you make them all kebab case?